x/chainlink/types/msg.go

```diff
@@ -86,7 +93,7 @@ func (m *ModuleOwner) GetSignBytes() []byte {
 }
 
 func (m *ModuleOwner) GetSigners() []sdk.AccAddress {
-	return []sdk.AccAddress{sdk.AccAddress(m.Address)}
+	return []sdk.AccAddress{sdk.AccAddress(m.AssignerAddress)}
```

Comment:
does the `assigner` become the `signer` of the newly added moduleowner then?

Reply:
yes, the assigner is always the signer for module owner tx who ideally should be one of the existing module owner